<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mqttSetting</class>
 <widget class="QWidget" name="mqttSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="nameLab">
              <property name="text">
               <string>用户名:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pwdLab">
              <property name="text">
               <string>密码:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="addrLab">
              <property name="text">
               <string>服务器地址:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="portLab">
              <property name="text">
               <string>端口号:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="idBtn">
              <property name="text">
               <string>clientID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="qosLab">
              <property name="text">
               <string>QoS:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="nameLineEdit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="pwdLineEdit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="addrLineEdit">
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="portLineEdit">
              <property name="text">
               <string>1883</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="idLineEdit"/>
            </item>
            <item>
             <widget class="QComboBox" name="qosBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="connectBtn">
            <property name="text">
             <string>开始连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="disconnectBtn">
            <property name="text">
             <string>断开连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
